code/modules/admin/sql_message_system.dm

```diff
@@ -428,10 +439,11 @@
 			var/server = query_get_type_messages.item[7]
 			var/editor_key = query_get_type_messages.item[8]
 			var/expire_timestamp = query_get_type_messages.item[9]
+			var/playtime = query_get_type_messages.item[10]
 			output += "<b>"
 			if(type == "watchlist entry")
 				output += "[t_key] | "
-			output += "[timestamp] | [server] | [admin_key]"
+			output += "[timestamp] | [server] | [admin_key] | [get_exp_format(text2num(playtime))]"
```

Comment:
I'd add a label of what this it's clear what it is i.e. `| 30h living playtime` instead of `| 30h`